### DISCUSSION TITLE
Is there a way to change the app's aspect ratio?

### DISCUSSION THREAD

**Discussion by christina-de-martinez** [callstack/react-native-visionos#129] [Q&A] [answered]
Another beginner question from me! 

I'm wondering if there's a way for me to control the aspect ratio of my app. If I initialize a normal React Native app, I can get it to work in a VisionOS simulator and it's roughly phone-shaped, in portrait mode. When I initialize a react-native-visionOS app per the instructions and open it in the simulator, it's a landscape rectangular shape. I'm wondering if I, as the dev, have any control over the sizing or aspect ratio of my app. For example, can I make a square shaped app? 

I'm a React dev and totally new to React Native, so this very well could be an anti-pattern. Please feel free to yell at me if that's the case, or redirect me if there's a better spot for this question.

**Comment by okwasniewski** [comment, marked as answer]
Hey! 

Yes, it's possible. You can achieve it by following this blog post: https://www.oskarkwasniewski.dev/blog/handling-window-resizing-in-visionos

**Reply by christina-de-martinez** [reply]
Incredible. Thank you so much!! 🔥
